packages/jni-swig-stub/src/main/jni/realm_api_helpers.cpp

```diff
@@ -1048,3 +1048,8 @@ realm_sync_thread_error(realm_userdata_t userdata, const char* error) {
     env->CallVoidMethod(static_cast<jobject>(userdata), java_callback_method, to_jstring(env, msg));
     jni_check_exception(env);
 }
+
+realm_scheduler_t*
+realm_create_generic_scheduler() {
+    return new realm_scheduler_t{realm::util::Scheduler::make_generic()};
```

Comment:
Hmm. Seems weird that the C-API offers an API around default schedulers that aren't necessarily support and does not have a API for this.

Reply:
Agree, it feels weird. I'll file an issue in Core for it

Reply:
But that one doesn't do anything, correct? Which seems to make it dangerous to use unless specifically requested?

Reply:
That's the goal, the generic one also don't do anything. They are mainly used for use case where we want to open a Realm without notifications  (like the Frozen Realms or when doing a compact). The dummy scheduler's implementation is slightly simpler that's all (both don't deliver anything)